frontend/public/components/cluster-settings/cluster-settings.tsx

```diff
@@ -56,9 +57,10 @@ const cancelUpdate = (cv: ClusterVersionKind) => {
 
 export const clusterAutoscalerReference = referenceForModel(ClusterAutoscalerModel);
 
-export const CurrentChannel: React.SFC<CurrentChannelProps> = ({cv}) => <button className="btn btn-link co-modal-btn-link" data-test-id="current-channel-update-link" onClick={() => clusterChannelModal({cv})}>
+export const CurrentChannel: React.SFC<CurrentChannelProps> = ({cv}) => <Button type="button" className="pf-m-inline" data-test-id="current-channel-update-link" onClick={() => clusterChannelModal({cv})} variant="link">
```

Comment:
Here and throughout:  use `isInline` instead of `className="pf-m-inline"`.  Apologies for not catching that earlier.

Reply:
updated